docs/dev/apis/lethallib/customscrap.md

```diff
@@ -0,0 +1,94 @@
+![Image of the audio mixer for diegetic audio.](/images/lethallib/customscrap/AudioMixMaster.png)
+
+### ScanNode Object
+On the ScanNode child gameobject, you should have the following:
+- Box collider tightly encompassing your model
+- Scan Node script
+
+The Scan Node script compontnet should have:
+- Max/Min range set to the max/min distance to have your scrap detectable (13, 1 are typical values)
+- Have require LOS enabled
+- Header text as the name to show when scanned
+- Sub Text set exactly to "Value: " (note the space after the colon)
+- Scrap Value should be 0 (this is modified by the game at runtime)
+- Creature Scan ID set to -1
+- Node Type set to 2
+The ScanNode object will also need to have the layer set to "ScanNode".
+
+After you've done this, you should go back to your Item Data and assign the Spawn Prefab to be the item prefab.
+
+After this, you should put your item data, prefab, any anything they rely on such as models into an asset bundle (See [Asset Bundling](/dev/intermediate/asset-bundling)). You can add multiple items to the same asset bundle if you're adding a collection of items.
+
+## Loading And Registering
+After creating your asset bundle and loading it in, you need to register your item prefab as a network object and add the scrap item data via LethalLib. This can be done with the following code:
+```cs
+int iRarity = 30;
+Item MyCustomItem = MyAssetBundle.LoadAsset<Item>("directory/to/itemdataasset.asset");
+LethalLib.Modules.NetworkPrefabs.RegisterNetworkPrefab(MyCustomItem.spawnPrefab);
+LethalLib.Modules.Items.RegisterScrap(MyCustomItem, iRarity, LethalLib.Modules.Levels.LevelTypes.All);
+```
+Rarity is a weight value that determines how likely the scrap is to spawn, with higher numbers meaning more likely to spawn. All scrap in the base game varies from 1 to 100 differing per-moon, and the value should be kept in that range.
+
+LevelTypes is a flag enum which provides some basic pre-defined options for moons, such as All, Vanilla, or any of the default moons individually. The enum supports bitwise operations, for example `(LethalLib.Modules.Levels.LevelTypes.March | LethalLib.Modules.Levels.LevelTypes.Dine)` will register the scrap for both March and Dine.
+
+There is a third optional parameter taking in an array of strings. Each string is compared to the internal moon name of all registered moons and adds the scrap to that moon if they match, allowing you to add scrap to custom moons. The names are mod-dependent; check with the custom moon author or check their source code for the name of the selectable level data asset for that moon. The vanilla moons can also be included through this string, where all vanilla moons' selectable level data follow the naming scheme "MoonnameLevel", like "ExperimentationLevel". It is recommended to follow this naming scheme if you intend on creating a custom moon for consistency.
```

Comment:
Consider splitting this paragraph up into two paragraphs - this article is more likely to be used by new modders and they will not be as enticed to read longer, continuous lines of text.